docs.md

```diff
@@ -126,7 +144,7 @@ Users can update the following fields if they have not been set, but after they
 
 In addition, as in the create validation, both a user subject and a group subject cannot be specified.
```

Comment:
Not part of this change per se, but I stumbled over it. Should be "`create` validation" (because 'create' is a standard English word).

Reply:
Better do it outside of this PR.